tee-worker/client-api/parachain-api/package.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "@litentry/parachain-api",
+    "name": "@heima/parachain-api",
     "type": "module",
     "license": "ISC",
     "main": "dist/src/index.js",
     "module": "dist/src/index.js",
     "sideEffects": false,
-    "version": "0.9.21-next.3",
```

Comment:
I think this should be a major version bump, it's a rebrand and has breaking changes